sqlalchemy-migrate.requirements

```diff
@@ -0,0 +1,8 @@
+decorator==4.0.10
```

Comment:
The idea is to have a separate virtualenv, which only used for DB migrations. There is no need to install kombu, etc for this. And at the same time there is no need for theses packages in the prod virtualenv.